GVRf/Sample/scene-objects/src/org/gearvrf/sample/sceneobjects/SampleViewManager.java

```diff
@@ -84,12 +84,13 @@ public void onInit(GVRContext gvrContext) {
         GVRSphereSceneObject sphereObject = new GVRSphereSceneObject(
                 gvrContext, true, material);
         GVRCylinderSceneObject cylinderObject = new GVRCylinderSceneObject(
-                gvrContext, 0.5f, 0.5f, 1.0f, 10, 36, true, futureTextureList, 2, 4);
+                gvrContext, true, material);
         GVRConeSceneObject coneObject = new GVRConeSceneObject(gvrContext,
                 true, material);
         GVRViewSceneObject webViewObject = createWebViewObject(gvrContext);
         GVRCameraSceneObject cameraObject = new GVRCameraSceneObject(
-                gvrContext, 8.0f, 4.0f, mActivity.getCamera());
+                gvrContext, mActivity, 3.6f, 2.0f, mActivity.getCamera());
```

Comment:
Yeah, somehow my eclipse project compiled with that bug. I did 'project clean' and rebuild and then I got the compile error too.